app/views/images/show.html.erb

```diff
@@ -1,8 +1,18 @@
 <h1><%= @image.title %></h1>
+<div>
+  <% if @image.likes.include?(user_id: current_user.id) %>
+    <%= link_to "UNLIKE", [@image, @image.likes.find_by(user_id: current_user.id)], method: :delete, class:'btn btn-primary' %>
```

Comment:
Technically links should only be used for GET requests. Rails does some magic to make the `method: :delete` thing work, but at the end of the day it's not ideal.

Instead, I would treat Liking and Unliking the same. Both are a form with a single button. One creates a like, the other destroys it.